CenterForOpenScience/isp
# Add indonesian translation

Refs: https://openscience.atlassian.net/browse/LEI-338

## Purpose
Add an indonesian translation

### TODO
- [x] Get site ID for consent form text
- [x] Make sure all text blocks are translated correctly
- [x] Verify capitalization in items left text

## Summary of changes
Add translation and consent forms

## Testing notes
- Create an account for the specified site: `INDO.ID`. Verify the correct consent form is shown.
- View the site with indonesian translation (Indonesia: Bahasa Indonesia)
